src/AdminPanel/modules/LoginPage/LoginPage.tsx

```diff
@@ -1 +1,41 @@
-export const LoginPage = () => <div data-cy="LoginPageDiv">Login page</div>;
+import { useForm, SubmitHandler } from 'react-hook-form';
+
+type LoginData = {
+  username: string;
+  password: string;
+};
+
+export const LoginPage = () => {
+  const {
+    register,
+    formState: { errors },
+    handleSubmit,
+    reset,
+  } = useForm<LoginData>({ mode: 'onTouched', defaultValues: { username: '', password: '' } });
```

Comment:
please create types.ts file and create enum for form values names

Reply:
resolved